app/javascript/app/providers/gdp-growth-provider/dummy.json

```diff
@@ -0,0 +1,70 @@
+{
+  "config": {
+    "axes": {
+      "xBottom": { "name": "Year", "unit": "date", "format": "YYYY" },
+      "yLeft": {
+        "name": "GDP Growth",
+        "unit": "",
```

Comment:
Maybe we should specify the percentage unit and prepare the table for this kind of data

Reply:
it's a good idea. what kind of a table do you mean?
I was also thinking about moving the `axis-ticks` files to maybe a common folder and have all of them grouped there with meaningful names, like `PercentageYAxis` etc.
Also for the tooltips.
WDYT?

Reply:
I think that could work well. We can reuse them for sure. Sorry, for table I meant graph.